package/Dockerfile

```diff
@@ -54,19 +54,19 @@ ENV CATTLE_PARTNER_CHART_DEFAULT_BRANCH=$PARTNER_CHART_DEFAULT_BRANCH
 ENV CATTLE_RKE2_CHART_DEFAULT_BRANCH=$RKE2_CHART_DEFAULT_BRANCH
 ENV CATTLE_HELM_VERSION v2.16.8-rancher2
 ENV CATTLE_MACHINE_VERSION v0.15.0-rancher114
-ENV CATTLE_K3S_VERSION v1.30.1+k3s1
+ENV CATTLE_K3S_VERSION v1.30.2+k3s2
 ENV CATTLE_MACHINE_PROVISION_IMAGE rancher/machine:${CATTLE_MACHINE_VERSION}
-ENV CATTLE_ETCD_VERSION v3.5.12
+ENV CATTLE_ETCD_VERSION v3.5.14
 ENV LOGLEVEL_VERSION v0.1.6
 ENV TINI_VERSION v0.18.0
 ENV TELEMETRY_VERSION v0.6.2
-ENV DOCKER_MACHINE_LINODE_VERSION v0.1.11
+ENV DOCKER_MACHINE_LINODE_VERSION v0.1.12
```

Comment:
@LBGarber , are you OK with bumping to `v0.1.12`? Looking at https://github.com/linode/docker-machine-driver-linode/releases/tag/v0.1.12 it's basically just dependency bumps but would like to get your 👍 on it.

Reply:
Bumping docker-machine-driver-linode to `v0.1.12` should be fine in a future PR 👍